layers/generate_layers.py

```diff
@@ -173,6 +178,30 @@ def fixProps(inputProps):
 
     convertGeoJsonToPmtiles(f"{tmp}/mrn.geojson", "output/mrn.pmtiles")
 
+def makeNationalCycleNetwork():
+    tmp = "tmp_ncn"
+    ensureEmptyTempDirectoryExists(tmp)
+
+    # Get the geojson from the link found at https://data-sustrans-uk.opendata.arcgis.com/
+    run(
+        [
+            "wget",
+            "https://opendata.arcgis.com/api/v3/datasets/5defd254e78745bfb12d0456abc1bcf1_0/downloads/data?format=geojson&spatialRefId=4326&where=1%3D1",
+            "-O",
+            f"{tmp}/national_cycle_network.geojson",
+        ]
+    )
+
+
+    def fixProps(inputProps):
+        propsToRemove = ["LinkNo", "Lighting",  "RoadClass", "SHAPE_Length", "GlobalID", "SegmentID"]
+        for prop in propsToRemove:
+            inputProps.pop(prop)
```

Comment:
I've never seen `pop` on a dictionary, only ever to remove the last thing from a list. But it works! TIL more python